examples/send_from_file/example.exs

```diff
@@ -111,32 +123,64 @@ defmodule Peer do
   end
 
   @impl true
-  def handle_info(:send_frame, state) do
-    Process.send_after(self(), :send_frame, 30)
+  def handle_info(:send_video_frame, state) do
+    Process.send_after(self(), :send_video_frame, 30)
 
-    case IVFReader.next_frame(state.ivf_reader) do
+    case IVFReader.next_frame(state.video_reader) do
       {:ok, frame} ->
-        {rtp_packets, payloader} = VP8Payloader.payload(state.payloader, frame.data)
+        {rtp_packets, payloader} = VP8Payloader.payload(state.video_payloader, frame.data)
 
         # the video has 30 FPS, VP8 clock rate is 90_000, so we have:
         # 90_000 / 30 = 3_000
-        last_timestamp = state.last_timestamp + 3_000 &&& @max_rtp_timestamp
+        last_timestamp = state.last_video_timestamp + 3_000 &&& @max_rtp_timestamp
 
         rtp_packets =
           Enum.map(rtp_packets, fn rtp_packet -> %{rtp_packet | timestamp: last_timestamp} end)
 
         Enum.each(rtp_packets, fn rtp_packet ->
-          PeerConnection.send_rtp(state.peer_connection, state.track_id, rtp_packet)
+          PeerConnection.send_rtp(state.peer_connection, state.video_track_id, rtp_packet)
         end)
 
-        state = %{state | payloader: payloader, last_timestamp: last_timestamp}
-        {:noreply, state}
+        {:noreply, %{state | video_payloader: payloader, last_video_timestamp: last_timestamp}}
 
       :eof ->
         Logger.info("video.ivf ended. Looping...")
-        {:ok, ivf_reader} = IVFReader.open("./video.ivf")
-        {:ok, _header} = IVFReader.read_header(ivf_reader)
-        state = %{state | ivf_reader: ivf_reader}
+        {:ok, reader} = IVFReader.open("./video.ivf")
+        {:ok, _header} = IVFReader.read_header(reader)
+        {:noreply, %{state | video_reader: reader}}
+    end
+  end
+
+  @impl true
+  def handle_info(:send_audio_packet, state) do
+    case OggReader.next_packet(state.audio_reader) do
+      {:ok, reader, {packet, duration}} ->
```

Comment:
The convention is to return reader at the last place in the result tuple. Take a look at Map.pop and others